### PR TITLE
Adjust to Gazebo 8 API and fix gazebo_ros_pkgs#612

### PR DESCRIPTION
**Pull request by mintar**

### CODE REVIEW

**Comment by mintar** [comment]
For a description, please see the extended individual commit messages. Tested on Gazebo 7 (the default in Kinetic) and Gazebo9.

**Comment by mintar** [comment]
Also see ros-simulation/gazebo_ros_pkgs#688 .

**Comment by costashatz** [comment]
@mintar thanks for the PR. I will try to give it a look as soon as possible. Thanks for your patience.

**Comment by mintar** [comment]
Sure, let me know if I can be of help.

File: src/mimic_joint_plugin.cpp [hunk 1]
```diff
@@ -22,21 +22,23 @@ OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISE
 
 #include <roboticsgroup_gazebo_plugins/mimic_joint_plugin.h>
 
+#if GAZEBO_MAJOR_VERSION >= 8
```
Comment (costashatz) [inline src/mimic_joint_plugin.cpp:25]: I would prefer something like (there's no actual need to use the whole namespaces):

```cpp
#if GAZEBO_MAJOR_VERSION >= 8
namespace math = ignition::math;
#else
namespace math = gazebo::math;
#endif
```

Then, you can use `math::clamp` in the code..

Comment (mintar) [reply src/mimic_joint_plugin.cpp:25]: Done!

Comment (costashatz) [reply src/mimic_joint_plugin.cpp:25]: Great! Thanks! I will test it in my machine and then accept it! Thanks again!

**Comment by costashatz** [comment]
@mintar thanks for the PR!

**Pull request merged**